contributing-device-support.md

```diff
@@ -1,23 +1,23 @@
-# Contributing New Board Support to Balena.io
+# Custom Board Support
 
-Pre-requisites: a [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible to the Yocto releases balenaOS supports.
+**Pre-requisites:** a [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible with the Yocto releases balenaOS supports.
 
-Repositories used to build balenaOS host Operating System (OS) are typically named balena-`<board-family>`. For example, consider [balena-raspberrypi](https://github.com/balena-os/balena-raspberrypi) which is used for building the OS for [Raspberryi Pi](https://raspberrypi.org), or [balena-intel][balena-intel repo] repository which can be used to build a balenaOS image for the Intel NUC boards.
+The repositories used to build the balenaOS host Operating System (OS) are typically called balena-`<board-family>`. For example, consider [balena-raspberrypi](https://github.com/balena-os/balena-raspberrypi) which is used for building the OS for [Raspberryi Pi](https://raspberrypi.org), or [balena-intel][balena-intel repo] repository which can be used to build a balenaOS image for the Intel NUC boards.
 
 Contributing support for a new board is a process that involves the following steps:
 
 * create the board support repository ([1](https://github.com/balena-os/meta-balena/blob/master/contributing-device-support.md#1-board-support-repository-breakout)),
-* because the board support repository needs to be located in balenaOS github organization, once the above steps have been done, 
-contact balena in order to have the repository moved / mirrored and the build system jobs created ([2](https://github.com/balena-os/meta-balena/blob/master/contributing-device-support.md#2-contact-balena)),
-* send a Pull Request for a hardware contract which describes the boards's capabilities ([3](https://github.com/balena-os/meta-balena/blob/master/contributing-device-support.md#3-hardware-contract)),
+* because the board support repository needs to be located in balenaOS GitHub organization, once the above steps have been done, 
```

Comment:
```suggestion
* because the board support repository needs to be located in the balena-os GitHub organization, once the above steps have been done, 
```